test/sparse/sparse.jl

```diff
@@ -1642,9 +1642,9 @@ end
 
 # Test temporary fix for issue #16548 in PR #16979. Brittle. Expect to remove with `\` revisions.
 # This is broken by the introduction of RowVector... see brittle comment above.
-#@testset "issue #16548" begin
-#    @test which(\, (SparseMatrixCSC, AbstractVecOrMat)).module == Base.SparseArrays
-#end
+@testset "issue #16548" begin
+    @test_broken which(\, (SparseMatrixCSC, AbstractVecOrMat)).module == Base.SparseArrays
```

Comment:
I fixed this for now, but I'm guessing that this will need to change again with `TransposedMatrix` (the thing it is testing may not even be true after that...).